simulation/costs.py

```diff
@@ -312,7 +312,7 @@ def estimate_boiler_upgrade_scheme_grant(
         return 0
 
     model_population_scale = ENGLAND_WALES_HOUSEHOLD_COUNT_2020 / model.household_count
-    boiler_upgrade_funding_cap_gbp = 450_000_000 / model_population_scale
+    boiler_upgrade_funding_cap_gbp = 250_000_000 / model_population_scale
```

Comment:
I'm not 100% sure how this change reflects the new announcement + your commit notes:

- The announcement you linked says: 
> The Secretary of State for Energy Security and Net Zero has given permission for Ofgem to over-allocate vouchers by up to £50 million in this financial year, to a total of up to £200 million, to provide certainty that vouchers will continue to be available for the remainder of the financial year.

 which I understood to be:
- total allowable to be allocated: £200M in financial year 1st Apr 2024-31st Mar 2025
- this is £50M above what was originally planned, i.e. previously only £150M allowed over same period

I think Ofgem's website is also saying the same thing as what I understood to be:
https://www.ofgem.gov.uk/environmental-and-social-schemes/boiler-upgrade-scheme-bus

I'm not sure where 250M has come from?

Reply:
If anything my interpretation of total cost of BUS is now:

- 2022 - 2023: £150M
- 2023 - 2024: £150M
- 2024 - 2025: £150M + £50M

Originally was £450M between 2022-2025, i.e. £150M per year.

Reply:
So simulation begins 1 Jan 2024, there is £200M in financial year 1st Apr 2024-31st Mar 2025 so I need to add on an amount 1Jan-1Apr 2024 = £200M / 4 = £50M

Reply:
Should it then be £150M / 4 though? 

Original grant was £450M over three years i think, i.e. £150M per year, and all the announcement is saying is £50M extra for 2024-2025, bringing total to £500M over 2022-2025?